src/reducers/filter.js

```diff
@@ -0,0 +1,12 @@
+// const initialState = 'ALL';
```

Comment:
You need to remove the commented code, It’s better to delete the commented code. If you need to review a previous version of your code, use git history.